### PR TITLE
Update README.md and remove last CVC4 references.

### PR DESCRIPTION
**Pull request by mpreiner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,31 +24,31 @@ cvc5 is intended to be an open and extensible SMT engine.  It can be used as a
...
-[COPYING](https://github.com/CVC4/CVC4/blob/master/COPYING)).
+[COPYING](https://github.com/cvc5/cvc5/blob/master/COPYING)).
 
 
 Website
 -------------------------------------------------------------------------------
-CVC4's website  is available at:
-http://cvc4.cs.stanford.edu/
+cvc5's website  is available at:
+https://cvc5.github.io/
```
Comment (yoni206) [inline README.md:33]: should we wait with this particular change until the website is updated? currently it says "coming soon".

File: README.md [hunk 2]
```diff
@@ -23,31 +24,31 @@ cvc5 is intended to be an open and extensible SMT engine.  It can be used as a
...
 
 Documentation
 -------------------------------------------------------------------------------
-Documentation for users of CVC4 is available at:
-http://cvc4.cs.stanford.edu/
+Documentation for users of cvc5 is available at:
+https://cvc5.github.io/docs/
```
Comment (yoni206) [inline README.md:38]: dito

**Review by yoni206 — APPROVED** [review]
LGTM with one minor question inline for your consideration before merging.

File: README.md (hunk 2 above)
Comment (mpreiner) [reply README.md:38]: This link actually already has some documentation.

File: README.md (hunk 1 above)
Comment (mpreiner) [reply README.md:33]: I think it's OK, the new website will be up and running soon.

Comment (yoni206) [reply README.md:33]: ditos crossed.

**Review by nafur — APPROVED** [review]
LGTM.

**Pull request merged**